script/cibuild

```diff
@@ -1,5 +1,7 @@
 #! /bin/sh
 
-bundle exec rspec
-bundle exec rubocop -S -D
-gem build jekyll-mentions.gemspec
```

Comment:
Do we not want to test that dependencies resolve and that the Gem builds?

Reply:
We do, default bundler rake task `bundle exec rake build` will take care of that.